pkg/receive/handler.go

```diff
@@ -12,18 +12,20 @@ import (
 	stdlog "log"
 	"net"
 	"net/http"
+	"path"
 	"sort"
 	"strconv"
 	"sync"
 	"time"
 
+	"github.com/mwitkow/go-conntrack"
+	"github.com/opentracing/opentracing-go"
+
```

Comment:
Accidentally changed formatting?

Reply:
damn, blame the IDE, I'll put it back on my next commit xD